packages/bento-design-system/src/defaultThemeClass.css.ts

```diff
@@ -0,0 +1,5 @@
+import { createTheme } from "@vanilla-extract/css";
```

Comment:
the `defaultTheme.css.ts` above is exporting the default theme as global, so that the user can just import `defaultTheme.css` and the style will be attached to the `:root` element.
If the user wants to use the `BentoThemeProvider` instead (i.e. creating a div with the theme className applied, for example to allow switching between different themes in the app), we also export the defaultTheme as a class (with `createTheme` instead of `createGlobalTheme`).